cluster-setup/ci-cluster/feature_sriov/10-sriov-daemonset.yaml

```diff
@@ -0,0 +1,66 @@
+kind: ConfigMap
```

Comment:
Please add a comment that says that this is a workaround for nics not officially supported by the operator.
In theory, if we manage to get a CI cluster with a supported nic, this could be removed because the operator will put the udev rule for us.

Also, if this doesn't end kustomized should be in base